### PR TITLE
Test both minimum supported and latest released libsodium

### PR DESCRIPTION
**Pull request by tarcieri**
Presently pinned to 1.0.0 and 1.0.3 respectively

@namelessjon I think this is what you were after...

### CODE REVIEW

**Comment by tarcieri** [comment]
Both libsodium 1.0.0 and 1.0.3 are working successfully on all versions we test on. Going to go ahead and merge this.

File: Gemfile [hunk 1]
```diff
@@ -9,7 +9,7 @@ end
 
 group :test do
   gem "coveralls", require: false
-  gem "rbnacl-libsodium"
+  gem "rbnacl-libsodium", ENV["LIBSODIUM_VERSION"]
```
Comment (tarcieri) [inline Gemfile]: Note that if `ENV["LIBSODIUM_VERSION"]` is unset, this just installs the latest version

**Comment by tarcieri** [comment]
Confirmed working here:

https://travis-ci.org/cryptosphere/rbnacl/jobs/71188526

```
Installing rbnacl-libsodium 1.0.0
```

and here:

https://travis-ci.org/cryptosphere/rbnacl/jobs/71188482

```
Installing rbnacl-libsodium 1.0.3
```

**Comment by namelessjon** [comment]
Great. Thanks for this

**Pull request merged**